src/Request.php

```diff
@@ -678,7 +678,7 @@ public function execute() {
 
 		}
 
-		return [];
+		return []; // @phpstan-ignore-line
```

Comment:
`$this->data` is defined as `OperationParams`, and `WPHelper::parseRequestParams` returns `OperationParams|OperationParams[]`, but since they're only defined in phpDoc and not in actual code, I'm wary to make the change without a second set of eyes.